Documentation/ApiOverview/Events/EventDispatcher/Index.rst

```diff
@@ -264,13 +276,15 @@ be written as:
     File; EXT:{extkey}/Configuration/Services.yaml
 ..  _EventDispatcherRegistration:
 
-Registering the event listener via :file:`Services.yaml`
---------------------------------------------------------
+Registering the event listener via :file:`Services.yaml` (legacy)
```

Comment:
This is not legacy, it is valid to register a listener that way (also in the future).

Reply:
Oh, ok. I thought this way was frowned upon. Changed.